tests/main/snap-confine-privs/task.yaml

```diff
@@ -0,0 +1,60 @@
+summary: ensure that snap-confine handles group and user privileges correctly
+details: |
+    The openSUSE security team has made a remark about a particular part of
+    snap-confine's UID/GID handling. The code there was, we believe, correct
+    but this test is here to demonstrate that and ensure it never regresses.
```

Comment:
The tests prove snap-confine has the desired behavior so we don't have to say 'we believe' here.